scripts/breakUpAriaJSON.js

```diff
@@ -117,6 +117,9 @@ fs.readFile(path.join('scripts/roles.json'), {
     for (let klass of klasses) {
       const newStack = stack.slice();
       newStack.unshift(klass);
+      if (aria[klass] === undefined) {
+        throw new TypeError(`Missing role '${klass}'`);
```

Comment:
Cool. @eps1lon I'm glad to see you're adventuring into the maintenance code of the project :)